samples/ComputeSharp.Sample.Shared/Helpers/Foo.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using ComputeSharp.__Internals;
+using ComputeSharp.D2D1Interop.__Internals;
+
+#pragma warning disable CS0618
+
+namespace ComputeSharp.D2D1Interop.__Internals
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [Obsolete("This interface is not intended to be used directly by user code")]
+    public interface ID2D1Shader
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be called directly by user code")]
+        void LoadDispatchMetadata<TLoader>(ref TLoader loader, out IntPtr result)
+            where TLoader : struct, IDispatchMetadataLoader;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be called directly by user code")]
+        void BuildHlslString(out ArrayPoolStringBuilder builder);
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be used directly by user code")]
+        void LoadBytecode<TLoader>(ref TLoader loader)
+            where TLoader : struct, IBytecodeLoader;
+    }
+}
+
+namespace ComputeSharp.D2D1Interop
+{
+    public interface ID2D1PixelShader : ID2D1Shader
+    {
+        float4 Execute();
+    }
+
+    
```

Comment:
did you accidentally extra whitespace?

Reply:
Uh... This whole file was just a local test, I accidentally committed it but it shouldn't be here 😆